pkg/operator/baremetal_pod.go

```diff
@@ -5,10 +5,23 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
+
+	"encoding/json"
+	"github.com/golang/glog"
+	"io/ioutil"
+	"math/rand"
+	"time"
+)
+
+const (
+	baremetalConfigmap = "metal3-config"
+	sharedVolume       = "metal3-shared"
```

Comment:
These names will be global to the operator package, so they should all be clearly identified as being related to the baremetal platform.

Reply:
Changed the names so that is clear that they are related to the baremetal platform.